test/CMakeLists.txt

```diff
@@ -15,6 +15,17 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+include(CheckIncludeFileCXX)
+CHECK_INCLUDE_FILE_CXX("tbb/version.h" HAVE_TBB)
+find_program(FIND_GXX_EXE g++)
+
+if (FIND_GXX_EXE)
+    execute_process(COMMAND ${FIND_GXX_EXE} -dumpversion OUTPUT_VARIABLE _onedpl_gxx_version)
+endif()
```

Comment:
Do you want to make test/CMakeLists.txt working without root CMakeLists.txt? Such scenarios are possible, but please carefully think about it: if you didn't mean such "standalone" use case then you don't need to initialize `_onedpl_gxx_version` here since you should already have it from the root CMakeLists.txt

Reply:
Using it as "standalone" is too specific, I can use `_onedpl_gxx_version` from root